data/daemon.conf

```diff
@@ -62,13 +62,13 @@ HostBkc=
 
 # The EFI system partition (ESP) path used if UDisks is not available
 # or if this partition is not mounted at /boot/efi, /boot, or /efi
-#EspLocation=
+EspLocation=
 
 # these are only required when the SMBIOS or Device Tree data is invalid or missing
-#Manufacturer=
-#ProductName=
-#ProductSku=
-#Family=
-#EnclosureKind=
-#BaseboardProduct=
-#BaseboardManufacturer=
+Manufacturer=
+ProductName=
+ProductSku=
+Family=
+EnclosureKind=
+BaseboardProduct=
+BaseboardManufacturer=
```

Comment:
this might need an associated code change I'm realizing; let me double check it.

Reply:
added matching code change for it.